### PR TITLE
Change the url for releases

### PR DESCRIPTION
**Pull request by popcor255**
Under the content/en/docs/Dashboard/_index.md page the url in the Important section was changed to have the correct url.
[Issue](https://github.com/tektoncd/website/issues/54)

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: content/en/docs/Dashboard/_index.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ kubectl apply --filename https://github.com/tektoncd/dashboard/releases/latest/d
 The latest release **may not be compatible** with your Tekton Pipelines
 installation, should you have an earlier release of Tekton Pipelines
 installed. For more compatibility information, see the
-[Tekton Dashboard Releases](https://github.com/tektoncd/dashboard/releases)
+[Tekton Dashboard Releases](https://github.com/tektoncd/dashboard#which-version-should-i-use)
 page.
```
Comment (ncskier) [inline content/en/docs/Dashboard/_index.md:49]: The link no longer points to the `Tekton Dashboard Releases` page, so we should probably change the text to match the new link.

Maybe something like the `Tekton Dashboard's "Which version should I use?" section`... or if that's too long, maybe the `Tekton Dashboard page`. (Or some other wording that makes more sense)

For reference, here are my suggested changes:

```suggestion
[Tekton Dashboard's "Which version should I use?"](https://github.com/tektoncd/dashboard#which-version-should-i-use)
section.
```

```suggestion
[Tekton Dashboard](https://github.com/tektoncd/dashboard#which-version-should-i-use)
page.
```

**Comment by popcor255** [comment]
@googlebot I consent.

**Comment by AlanGreene** [comment]
@ncskier looks like you need to consent to being added as co-author

**Comment by ncskier** [comment]
@googlebot I consent

**Comment by ncskier** [comment]
@googlebot I consent.

**Comment by ncskier** [comment]
@googlebot I consent.

**Comment by ncskier** [comment]
I really don't have to be added as a co-author if the googlebot CLA stuff isn't working 😅

**Comment by skaegi** [comment]
/approve
Thanks!!

**Pull request merged**